test/irc_test.rb

```diff
@@ -291,6 +291,14 @@ def test_no_colors
     refute_match /\003/, privmsg
   end
 
+  def test_include_repo_name_and_owner_in_irc_realname
+    svc = service(:pull_request, {'room' => 'r', 'nick' => 'n'}, pull_payload)
+
+    svc.receive_pull_request
+    msgs = svc.writable_irc.string.split("\n")
+    assert_includes msgs, "USER n 8 * :GitHub IRCBot - defunkt/grit"
```

Comment:
In the [payload used in the tests](https://github.com/github/github-services/blob/8ac127e018e2b8eedc14e3d823605cc2fbcfff1a/lib/service/events/helpers/helpers_with_meta.rb#L12-L16), the repository is "mojombo/grit". "defunkt" is the sender, not the repository owner. I think you want your assertion to look like [this](https://github.com/github/github-services/compare/2438ee27c35af9c16f3de31a1c702fa3caa5cf3f...8ac127e018e2b8eedc14e3d823605cc2fbcfff1a#diff-c5ef582e0e679b233de71e8fa43f5254R299).

```ruby
assert_includes msgs, "USER n 8 * :GitHub IRCBot - mojombo/grit"
```

And to make that work, I think you want the implementation to look more like [this](https://github.com/github/github-services/compare/2438ee27c35af9c16f3de31a1c702fa3caa5cf3f...8ac127e018e2b8eedc14e3d823605cc2fbcfff1a#diff-df7cb3c7def6051de45c5de7d5b4b8a8R144):

```ruby
def irc_realname
  repo_owner = payload["repository"]["owner"]["name"]
  repo_name = payload["repository"]["name"]

  "GitHub IRCBot - #{repo_owner}/#{repo_name}"
end
```

What do you think?

Feel free to cherry-pick any of [these commits](https://github.com/github/github-services/compare/2438ee27c35af9c16f3de31a1c702fa3caa5cf3f...8ac127e018e2b8eedc14e3d823605cc2fbcfff1a#diff-c5ef582e0e679b233de71e8fa43f5254R299) if you like.